File: module_beauty/src/main/java/com/gas/beauty/article/mvp/ArticleContract.java
package com.gas.beauty.article.mvp;

import android.content.Context;

import androidx.fragment.app.Fragment;

import com.base.lib.mvp.IView;

public interface ArticleContract {


    interface View extends IView {

        Context getWrapContext();
        Fragment getCurrentFragment();

    }

    interface Model {

    }


}

File: module_beauty/src/main/java/com/gas/beauty/ui/article/mvp/ArticleContract.kt
package com.gas.beauty.ui.article.mvp

import android.content.Context
import androidx.fragment.app.Fragment
import com.base.lib.mvp.IView

interface ArticleContract {
    interface View : IView {
        val wrapContext: Context
        val currentFragment: Fragment
    }

    interface Model
}